Minor doc clarifications about upgrading to a new version of MRTK (#6935)
microsoft/MixedRealityToolkit-Unity: Documentation/ReleaseNotes.md

For the smoothest upgrade path, please use the following steps.

1. Close Unity

> 1. Inside the Assets folder, delete most of the **MixedRealityToolkit** folders (the project may not have all listed folders)

Reviewer: ```suggestion
1. Inside the *Assets* folder, delete most of the **MixedRealityToolkit** folders (the project may not have all listed folders)
```

Reviewer: and add ", or its .meta file.: to "Do NOT delete the generated folder"

1. Close Unity and Delete the **Library** folder. This step is necessary to force Unity to refresh its
   asset database and reconcile existing custom profiles.
1. Launch Unity, and for each scene in the project

>     - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the hierarchy. This will delete the main camera, but it will be re-created in the next step. If you've changed any properties of the main camera, you may want to note those changes and re-apply them.

Reviewer: ```suggestion
    - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the hierarchy. This will delete the main camera, but it will be re-created in the next step. If any properties of the main camera have been manually changed, these will have to be re-applied manually once the new camera is created.
```

Reviewer: Avoid use of "you". I know a lot of MRTK documentation does not entirely follow this, but trying to move away from the informal

Author: TBH, while I agree with being formal, I disagree that "you" is informal.  I think it is important to avoid passive voice, too.  However I'll do as you've asked.

- Windows 10 SDK 18362 or later (installed by the Visual Studio Installer)
- Unity 2018.4, 2019.1 or 2019.2

### How to upgrade projects to 2.0.0

Since the RC2 release, there have been several changes that may impact application projects,
including some files moving to new folder locations. Breaking change details, including mitigation guidance, can be found in the [**Updating RC2 to 2.0.0**](Updating.md#updating-rc2-to-200) article.

For the smoothest upgrade path, please use the following steps.

1. Close Unity

> 1. Inside the Assets folder, delete most of the **MixedRealityToolkit** folders, along with their .meta files (the project may not have all listed folders)

Reviewer: ```suggestion
1. Inside the *Assets* folder, delete most of the **MixedRealityToolkit** folders, along with their .meta files (the project may not have all listed folders)
```

1. Close Unity and Delete the **Library** folder. This step is necessary to force Unity to refresh its
   asset database and reconcile existing custom profiles.
1. Launch Unity, and for each scene in the project

>     - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the hierarchy. This will delete the main camera, but it will be re-created in the next step. If you've changed any properties of the main camera, you may want to note those changes and re-apply them.

Reviewer: ```suggestion
    - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the hierarchy. This will delete the main camera, but it will be re-created in the next step. If any properties of the main camera have been manually changed, these will have to be re-applied manually once the new camera is created.
```
